### PR TITLE
Add spellrelax and apertium-separable closes #23, #29, #58, #62

### LINKED ISSUE #23: support for more features in hfst modules
Add optional support to hfst modules for enabling spelling modules, an extra [twoc module for morphotactic constraints](http://wiki.apertium.org/wiki/Morphotactic_constraints_with_twol), and spellrelax.  Perhaps these should all be options at configure/autogen time, so that they can be enabled as needed after a module is started, but skeletons of the files they require should probably also be provided.

### PR DESCRIPTION
**Pull request by mr-martian**
I'm not completely certain of the naming of `spellrelax.twol`. @jonorthwash

### CODE REVIEW

**Comment by mr-martian** [comment]
@jonorthwash said spellrelax is generally done with `.twoc`, so I've changed it to `spellrelax.twoc`.

**Comment by ftyers** [comment]
@mr-martian that is not correct, current spellrelax is done with .twol files.

**Comment by mr-martian** [comment]
@ftyers Should I change it back then or would it be better to give it a name more like the other files? (`apertium-abc.spellrelax.twol` or something)

**Comment by ftyers** [comment]
Something like `apertium-abc.abc.spellrelax` ?

**Comment by mr-martian** [comment]
Ok, default spellrelax file is now named `apertium-abc.abc.spellrelax`. It is twol and does apostrophe unification.

File: README.md [hunk 1]
```diff
@@ -45,6 +45,18 @@ To bootstrap and compile it at the same time,
 $ python3 apertium-init.py foo --analyser=hfst && ( cd apertium-foo/ && ./autogen.sh && make )
 ```
 
+To include a twoc file for handling prefixes,
```
Comment (jonorthwash) [inline README.md]: Didn't we say it'd be `.spellrelax` ?

File: README.md [hunk 2]
```diff
@@ -45,6 +45,18 @@ To bootstrap and compile it at the same time,
...
+
+```bash
+$ python3 apertium-init.py foo --analyser=hfst --with-twoc
```
Comment (jonorthwash) [inline README.md]: See above.

File: README.md (hunk 1 above)
Comment (mr-martian) [reply README.md]: I'm not sure what you're referring to. This example is for the `--with-twoc` option.

Comment (jonorthwash) [reply README.md]: Ah, sorry, I was getting the two twoc files confused.

**Comment by sushain97** [comment]
Great work!

**Pull request merged**